integration-tests/src/test/resources/sitconfig/configoverrides/config.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
missing copyright notice

Reply:
Added.

Reply:
The server doesn't like the comments in configuration override files.
Configuration override fails with the following error.


<Mar 22, 2019 4:57:45 AM GMT> <Notice> <WebLogicServer> <BEA-000365> <Server state changed to STARTING.>
<Mar 22, 2019 4:57:45 AM GMT> <Warning> <Management> <BEA-141323> <The situational config file has an invalid format, it is being ignored: XMLSituationalConfigFile[/shared/domains/customsitconfigdomain/optconfig/diagnostics/WLDF-MODULE-0-9771-situational-config.xml] because org.xml.sax.SAXParseException; lineNumber: 4; columnNumber: 6; The processing instruction target matching "[xX][mM][lL]" is not allowed.
org.xml.sax.SAXParseException; lineNumber: 4; columnNumber: 6; The processing instruction target matching "[xX][mM][lL]" is not allowed.
        at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:203)
        at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:177)

Reply:
Because of the errors I had to remove the copyright notice in config override files.

Reply:
ok fair enough